source/gui/settingsDialogs.py

```diff
@@ -1111,12 +1111,27 @@ def makeSettings(self, settingsSizer):
 		self.lineNumberCheckBox=wx.CheckBox(self,wx.NewId(),label=_("Report line &numbers"))
 		self.lineNumberCheckBox.SetValue(config.conf["documentFormatting"]["reportLineNumber"])
 		settingsSizer.Add(self.lineNumberCheckBox,border=10,flag=wx.BOTTOM)
-		# Translators: This message is presented in the document formatting settings dialogue
-		# If this option is selected, NVDA will cound the leading spaces and tabs of a line and speak it.
-		#
-		self.lineIndentationCheckBox=wx.CheckBox(self,wx.NewId(),label=_("Report l&ine indentation"))
-		self.lineIndentationCheckBox.SetValue(config.conf["documentFormatting"]["reportLineIndentation"])
-		settingsSizer.Add(self.lineIndentationCheckBox,border=10,flag=wx.BOTTOM)
+		sizer=wx.BoxSizer(wx.HORIZONTAL)
+		# Translators: This is the label for a combobox in the
+		# Document  Formatting  dialog (possible choices are Off, Speech, Tones, or Both.
+		sizer.Add(wx.StaticText(self,wx.ID_ANY,label=_("Report Line &Indents With:")))
+		indentChoices=[
+			#Translators: A setting to report No  line Indentation.
+			_("Off"),
+			#Translators: A setting to report indents with Speech.
+			_("Speech"),
+			#Translators: A setting to report indents with tones.
+			_("Tones"),
+			#Translators: A setting to report indents with both tones and Speech.
+			_("Both  Speech and Tones")
+		]
+		self.lineIndentationCombo = wx.Choice(self,wx.ID_ANY,choices=indentChoices)
+		#This is for Speech. For legacy reasons, we leave this name.

```

Comment:
I'm guessing this comment refers to the reportLineIndentation key, but given that the code below refers to both keys, this is unclear. Please clarify or remove.